Kitodo/src/main/webapp/pages/PasswortAendern.xhtml

```diff
@@ -92,25 +92,25 @@
 
 
                                                         <!-- neues passwort -->
-                                                        <h:outputLabel id="id11" for="passwortAendernNeu1"
```

Comment:
Changes on the frontend are not advisable right now, as the whole frontend is in the replacement.

Reply:
The field name contains a digit which means that the corresponding getters and setters contain a digit, which is illegal to checkstyle: Method names regex forbids numbers in getters and setters. So field names must not contain any of them. We should rethink if we want to set ourself rules that we do not want to follow.

Reply:
If you change the method name in the form class then you should change it in the xhtml as we need a working application.